scipy/optimize/_differentialevolution.py

```diff
@@ -1044,8 +1045,8 @@ def __next__(self):
             self._promote_lowest_energy()
 
         if self.dither is not None:
-            self.scale = (self.random_number_generator.rand()
-                          * (self.dither[1] - self.dither[0]) + self.dither[0])
+            self.scale = self.random_number_generator.uniform(self.dither[0],
+                                                              self.dither[1])
```

Comment:
should this continue over so far in the line?

Reply:
Change is PEP8 compliant for line length.